anvil_consortium_manager/adapters/workspace.py

```diff
@@ -131,6 +131,11 @@ def get_autocomplete_queryset(self, queryset, q, forwarded=None):
             queryset = queryset.filter(workspace__name__icontains=q)
         return queryset
 
+    def get_extra_detail_context_data(self, extra_context={}):
```

Comment:
I think this method should also take the request as a required argument. This way, someone could check if the user had specific permissions and then do something different based on permissions.